docs/tutorial/2-deploy-your-first-contract.md

```diff
@@ -20,7 +20,7 @@ This guide will walk you through deploying and interacting with contracts in REM
    1. Using Talisman wallet
 
       Select the **Injected Provider - Talisman** environment in the **Deploy & Run** tab. When prompted, allow Remix to connect to Talisman.
-      Switch to the **Westend Asset Hub** network in Talisman. Make sure **Enable Testnets** is checked in Talisman to see the network.
```

Comment:
There is not such network like Westend in Talisman, there is only Westend Asset Hub. Do we want to rename it in substrate based Wallets? The same here: https://chainlist.org/?search=westend